src/components/Home.jsx

```diff
@@ -0,0 +1,43 @@
+import { useSelector } from 'react-redux/es/hooks/useSelector';
+import { Link } from 'react-router-dom';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { fetchcategories } from '../redux/CategoriesSlice';
+import Header from './Header';
+
+const Home = () => {
+  const category = useSelector((state) => state.category.category.data);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (category) dispatch(fetchcategories());
+  }, []);
+
+  if (category) {
+    return (
+      <div>
+        <Header />
+        <ul className="category-list">
+          {category.map((m) => (
+            <Link to={`/games/${m.id}`} key={m.id} className="link">
+              <li className="list">
```

Comment:
- [ ] As previously requested, please add an input field that allows users to filter cities by typing their names. 🖋️😊